src/app/public/modules/date-range-picker/date-range-calculator-config.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  SkyDateRangeCalculatorGetValueFunction
+} from './date-range-calculator-date-range-function';
+
+import {
+  SkyDateRangeCalculatorType
+} from './date-range-calculator-type';
+
+import {
+  SkyDateRangeCalculatorValidateFunction
+} from './date-range-calculator-validate-function';
+
+export interface SkyDateRangeCalculatorConfig {
```

Comment:
Should we move these interfaces into a `/types` folder like our other repos? I see a couple that could fall into that category:

- `SkyDateRangeCalculatorConfig`
- `SkyDateRangeDefaultCalculatorConfig`
- `SkyDateRangeCalculatorType`
- `SkyDateRange`
etc...

Reply:
Yeah, I suppose. I've grown to dislike the `types` name because technically everything in TypeScript is a "type". I'll make the change for consistency's sake, though!